### PR TITLE
MBS-7873 Update android lint documentation

### PR DESCRIPTION
**Pull request by eugene-krivobokov**

### CODE REVIEW

File: docs/content/docs/ci/CIGradlePlugin.md [hunk 1]
```diff
@@ -38,57 +38,143 @@ builds {
 - fullCheck - максимально полные проверки, можем жертвовать скоростью
 - release - набор действий необходимых для предоставления всех релизных артефактов
 
-### Steps
+## Steps
```
Comment (eugene-krivobokov) [inline docs/content/docs/ci/CIGradlePlugin.md:41]: Screenshot from 2020-03-12 17-00-33

File: docs/content/docs/checks/AndroidLint.md [hunk 2]
```diff
@@ -5,34 +5,42 @@ type: docs
...
+We use [Android lint](https://developer.android.com/studio/write/lint) to check applications in CI.
 
-В PR результаты отображаются во вкладках "\<app name\> lint".
+By default, we check only applications. It's no use to check libraries in isolation.
 
-## Настройки
+## How to run lint locally
 
-Помимо стандартных настроек lint у нас есть:
+`./gradlew :app:lintRelease`
```
Comment (sergioboi) [inline docs/content/docs/checks/AndroidLint.md]: It is not clear what project command can works. 
Change `app` to `app-module-name` or mark page internal

File: docs/content/docs/checks/AndroidLint.md [hunk 3]
```diff
@@ -5,34 +5,42 @@ type: docs
...
-- Проставляем общие lintOptions для всех модулей в рутовом build.gradle
-- Рутовый lint.xml для задания исключений для всех модулей в репозитории
-- Плагин для общего lint отчета по всем модулям: `com.avito.android.lint-report`
+## Configure lint to suppress warnings
 
-## Как подавить ошибку?
+You can use default capabilities of Android lint ([Configure lint to suppress warnings](https://developer.android.com/studio/write/lint.html#config)). 
 
-Используем стандартные возможности lint ([Configure lint to suppress warnings](https://developer.android.com/studio/write/lint.html#config)): 
+It's better to minimize a scope of suppressing:
```
Comment (sergioboi) [inline docs/content/docs/checks/AndroidLint.md]: Explain please why better

File: docs/content/docs/checks/AndroidLint.md [hunk 4]
```diff
@@ -5,34 +5,42 @@ type: docs
...
-- Добавить исключение в lint.xml
-    - Конкретного модуля (android library).\
-    Используем lintOptions.checkDependencies, поэтому может не примениться для приложения, которое подключает модуль. 
-    - Приложения (android application)
-    - Всего репозитория. Отключит проверку во всех модулях.
+- `@SuppressLint` annotation in the code
+- `tools:ignore` attribute in XML files
 
-## Custom lint checks
+Otherwise, use `lint.xml` config file in an application module.
```
Comment (sergioboi) [inline docs/content/docs/checks/AndroidLint.md]: Explain otherwise

File: docs/content/docs/checks/AndroidLint.md [hunk 5]
```diff
@@ -5,34 +5,42 @@ type: docs
...
+
+## Configure lint to run in CI
+
+Add a `lint` [build step]({{< ref "/docs/ci/CIGradlePlugin.md#android-lint-step" >}}) to a build in `build.gradle`
+
+```groovy
+fastCheck {
```
Comment (sergioboi) [inline docs/content/docs/checks/AndroidLint.md:33]: It looks like magic. Do we need to explain that fast check runs on ci? What is fastCheck?

File: docs/content/docs/ci/CIGradlePlugin.md [hunk 6]
```diff
@@ -38,57 +38,143 @@ builds {
...
 
-Внутри сценариев описываем декларативно необходимые шаги:
+Step is a declaration to run some logic. It works inside a scenario:
```
Comment (sergioboi) [inline docs/content/docs/ci/CIGradlePlugin.md:43]: It's not clear that unitTests, lint, uiTests are steps

File: docs/content/docs/ci/CIGradlePlugin.md [hunk 7]
```diff
@@ -38,57 +38,143 @@ builds {
...
+
+```groovy
+uploadBuildResult {
+    uiTestConfiguration = "regression" // instrumentation configuration
+}
+```
+
+#### Deploy to Google Play
+
+Deploy to Google play.
```
Comment (sergioboi) [inline docs/content/docs/ci/CIGradlePlugin.md:163]: Deploy works with the config. Mark step internal

**Review by sergioboi — CHANGES_REQUESTED** [review]

File: docs/content/docs/checks/AndroidLint.md [hunk 8]
```diff
@@ -5,34 +5,42 @@ type: docs
 
 # Android Lint
 
-https://developer.android.com/studio/write/lint
+We use [Android lint](https://developer.android.com/studio/write/lint) to check applications in CI.
 
-В PR результаты отображаются во вкладках "\<app name\> lint".
+By default, we check only applications. It's no use to check libraries in isolation.
```
Comment (int02h) [inline docs/content/docs/checks/AndroidLint.md]:
```suggestion
By default, we check only applications. It's not use to check libraries in isolation.
```

File: docs/content/docs/ci/CIGradlePlugin.md [hunk 9]
```diff
@@ -38,57 +38,143 @@ builds {
...
+
+```groovy
+unitTests {}
+```
+
+#### Upload to QApps
+
+{{< hint info>}} This section contains Avito specific information {{< /hint >}}
+
+Upload [artifacts]({{< relref "#collecting-artifacts">}}) to QApps (internal system )
```
Comment (int02h) [inline docs/content/docs/ci/CIGradlePlugin.md]:
```suggestion
Upload [artifacts]({{< relref "#collecting-artifacts">}}) to QApps (internal system)
```

File: docs/content/docs/checks/AndroidLint.md (hunk 8 above)
Comment (eugene-krivobokov) [reply docs/content/docs/checks/AndroidLint.md]: I mean useless: https://idioms.thefreedictionary.com/it%27s+no+use

File: docs/content/docs/checks/AndroidLint.md (hunk 2 above)
Comment (eugene-krivobokov) [reply docs/content/docs/checks/AndroidLint.md]: fixed

File: docs/content/docs/checks/AndroidLint.md (hunk 3 above)
Comment (eugene-krivobokov) [reply docs/content/docs/checks/AndroidLint.md]: fixed

File: docs/content/docs/checks/AndroidLint.md (hunk 4 above)
Comment (eugene-krivobokov) [reply docs/content/docs/checks/AndroidLint.md]: Rephrased

File: docs/content/docs/checks/AndroidLint.md (hunk 5 above)
Comment (eugene-krivobokov) [reply docs/content/docs/checks/AndroidLint.md:33]: Added an internal marker to the page

File: docs/content/docs/ci/CIGradlePlugin.md (hunk 6 above)
Comment (eugene-krivobokov) [reply docs/content/docs/ci/CIGradlePlugin.md:43]: I've commented in the example. 
I think it's enough here cause we have a detailed explanation below.

File: docs/content/docs/ci/CIGradlePlugin.md (hunk 9 above)
Comment (eugene-krivobokov) [reply docs/content/docs/ci/CIGradlePlugin.md]: fixed

File: docs/content/docs/ci/CIGradlePlugin.md (hunk 7 above)
Comment (eugene-krivobokov) [reply docs/content/docs/ci/CIGradlePlugin.md:163]: fixed

**Review by sergioboi — APPROVED** [review]

**Pull request merged**